src/utilities.ts

```diff
@@ -292,7 +292,7 @@ export async function getAddressBalance(network: string, address: string) {
       locked: lockedBalance.toString(),
       unlocked: unlocked.toString(),
     };
-  } catch (error) {
+  } catch (error : any) {
     throw new Error(`Failed to get address balance: ${error.message}`);
   }
```

Comment:
```suggestion
  } catch (error) {
    if (error instanceof Error) {
      throw new Error(`Failed to get address balance: ${error.message}`);
    } else {
      throw new Error(`Failed to get address balance: ${String(error)}`);
    } 
  }
```

Reply:
Applying the format 
I just extracted it a utility function as this may be reused going on